lakefile.lean

```diff
@@ -10,7 +10,7 @@ open Lake DSL
 require "leanprover-community" / "batteries" @ git "main"
 require "leanprover-community" / "Qq" @ git "master"
 require "leanprover-community" / "aesop" @ git "master"
```

Comment:
Should tagged releases be pinned to other tagged releases?

Reply:
No, we decided that was too annoying, as it requires then untagging them in a subsequent PR so that the automatic update bot works again.

Reply:
I don't think it's necessary to coordinate all the `v4.X.0` tags across the ecosystem precisely. As long as they all exist, and depend on something no earlier than the `v4.X.0` tag of each project upstream, it should be fine.